influxdb_client/client/_base.py

```diff
@@ -99,7 +99,9 @@ def __init__(self, url, token, debug=None, timeout=10_000, enable_gzip=False, or
         pass
 
     @classmethod
-    def _from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gzip=False, **kwargs):
+    def _from_config_file(cls, config_file: str = "config.ini", debug=None,
+                          config_name: str = "influx2", enable_gzip=False, **kwargs):
+
```

Comment:
Please use `kwargs` instead of new argument:

```suggestion
    def _from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gzip=False, **kwargs):

```